Games/xboxfeed.10m.js

```diff
@@ -3,16 +3,17 @@
 
 /*
 <bitbar.title>Xbox Feed</bitbar.title>
-<bitbar.version>v1.0.1</bitbar.version>
+<bitbar.version>v1.1.0</bitbar.version>
 <bitbar.author>Kodie Grantham</bitbar.author>
 <bitbar.author.github>kodie</bitbar.author.github>
 <bitbar.desc>Shows your Xbox Live friend's recent activity - Be sure to read the installation instructions here: https://github.com/kodie/bitbar-xboxfeed</bitbar.desc>
 <bitbar.image>https://raw.githubusercontent.com/kodie/bitbar-xboxfeed/master/screenshot.png</bitbar.image>
-<bitbar.dependencies>node, npm, npm/https://github.com/y-a-v-a/easy-gd.git, npm/fs, npm/home-config, npm/node-time-ago, npm/request, npm/sync-request</bitbar.dependencies>
+<bitbar.dependencies>node, npm, npm/deasync, npm/home-config, npm/jimp, npm/node-time-ago, npm/open, npm/request</bitbar.dependencies>
+<bitbar.dependencies.npm>npm/deasync, npm/home-config, npm/jimp, npm/node-time-ago, npm/open, npm/request</bitbar.dependencies>
```

Comment:
This isn't a normal metadata tag, but that should be fine, but you need to make sure the closing tag is correct.  Should be `</bitbar.dependencies.npm>`

```
ERR! /Users/travis/build/matryer/bitbar-plugins/Games/xboxfeed.10m.js includes mismatched metatags: <bitbar.dependencies.npm>npm/deasync, npm/home-config, npm/jimp, npm/node-time-ago, npm/open, npm/request</bitbar.dependencies>
```